src/include/platform/internal/GenericPlatformManagerImpl_FreeRTOS.ipp

```diff
@@ -215,6 +215,14 @@ void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::_RunEventLoop(void)
             eventReceived = xQueueReceive(mChipEventQueue, &event, 0);
         }
     }
+
+#if defined(INCLUDE_vTaskDelete) && INCLUDE_vTaskDelete
+    if (mEventLoopTask)
+    {
+        vTaskDelete(mEventLoopTask);
+        mEventLoopTask = nullptr;
+    }
```

Comment:
You need to handle the static task alloc case. See `CHIP_ERROR GenericPlatformManagerImpl_FreeRTOS<ImplClass>::_StartEventLoopTask(void)` and the `CHIP_CONFIG_FREERTOS_USE_STATIC_TASK` define